src/typescript/models/todoModel.ts

```diff
@@ -0,0 +1,21 @@
+import Category from './categoryModel';
+
+class Todo {
```

Comment:
Why use class?

Reply:
I use this class because when I expand to connect the two databases, it will look like this **_in the picture_**. When I reconnect via Id Category, the Todo Object will have another property, Category Object, so I use Class Category in the Todo class @phuocduong-agilityio.
![image](https://github.com/HoangLiarPhuongLe/typescript-training/assets/110210274/e84a476c-e324-4b1a-9ad7-a3d3b9a82709)